src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1295,6 +1295,53 @@ namespace DataSurfaces {
         }
     };
 
+    // Surface interior convection
+    struct SurfIntConv {
```

Comment:
Here are the new surface convection structs. @mjwitte, please advise on field names.